### PR TITLE
Bower => NPM

### PR DESCRIPTION
**Pull request by acburdine**
Supersedes #847, requires #969 

This PR will remove all bower dependencies, swapping out the ones we can for npm dependencies and removing ones that aren't used at all.

## Tasks
- [x] Remove validator & clean up validator usage
- [x] Remove remainder of bower components
- [x] clean up node-assets imports
- [x] After #969 is merged, rebase & remove bower dep.

### CODE REVIEW

**Comment by acburdine** [comment]
now that #969 is merged i'll clean this up and get it working completely.

**Comment by kevinansfield** [comment]
👍 thanks for looking into this, it's been on the back burner for far too long! 😄

File: package.json [hunk 1]
```diff
@@ -118,6 +118,7 @@
     "simplemde": "https://github.com/kevinansfield/simplemde-markdown-editor.git#ghost",
     "testem": "2.0.0",
     "top-gh-contribs": "2.0.4",
+    "validator": "7.2.0",
```
Comment (acburdine) [inline package.json]: validator's latest version is technically 9.x, however there was a breaking change in 8.x that changed how the isURL function worked with localhost urls. Probably could update it to 9.x, but that should be a separate pr IMO, as the usages of isURL will need to be evaulated. (For what it's worth, Ghost-CLI is stuck on validator 7.2 for the same reason)

Comment (kevinansfield) [reply package.json]: 👍

**Comment by kevinansfield** [comment]
Pushed a rebase to fix yarn.lock conflicts
